src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/BlazorDebugConfigurationProvider.ts

```diff
@@ -0,0 +1,91 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { spawn } from 'child_process';
+import * as vscode from 'vscode';
+
+import { RazorLogger } from '../RazorLogger';
+import { onDidTerminateDebugSession } from './Events';
+
+export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurationProvider {
+
+    constructor(private readonly logger: RazorLogger, private readonly vscodeType: typeof vscode) { }
+
+    public async resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, configuration: vscode.DebugConfiguration): Promise<vscode.DebugConfiguration | undefined> {
+        const output = this.vscodeType.window.createOutputChannel('Blazor WebAssembly App');
+        const app = spawn('dotnet', ['run'], {

```

Comment:
>const [](start = 8, length = 5)

Do you know if stdout might contain unicode characters? If so, on Windows, you probably want `cmd.exe /c CHCP 65001 & dotnet run` to force the console code page to UTF-8.

Reply:
Yes, there's a chance that a user maybe running in a non-English locale in which case we will need to support unicode characters. Good note!